rs/bitcoin/kyt/btc_kyt_canister.did

```diff
@@ -46,7 +46,24 @@ type CheckTransactionIrrecoverableError = variant {
     InvalidTransaction : text;
 };
 
-service : {
+type InitArg = record {
+    network : Network
+};
+
+type UpgradeArg = record {};
+
+type KytArg = variant {
+    InitArg : InitArg;
+    UpgradeArg: UpgradeArg; 
+};
```

Comment:
supra nit: in case we want to be somewhat consistent with the ckBTC minter, which has `MinterArg`, but `InitArgs` (plural) and `UpgradeArgs` (plural)

```suggestion
type KytArg = variant {
    Init : InitArgs;
    Upgrade: opt UpgradeArgs; 
};
```
Reagrdless of the potential name change in the types, I would make the `Upgrade` parameter optional (as in the above suggestion).